plugins/NitroSpoof/src/main/java/com/xinto/aliuplugins/NitroSpoof.kt

```diff
@@ -66,14 +66,12 @@ class NitroSpoof : Plugin() {
         finalUrl += idStr
         val emoteSize = settings.getString(EMOTE_SIZE_KEY, EMOTE_SIZE_DEFAULT).toIntOrNull()
 
-        finalUrl += if (isAnimated) ".gif" else ".png"
+        finalUrl += if (isAnimated) ".gif" else ".png" + "?quality=lossless"
```

Comment:
This would need to be put in parenthesis, like this:
```kt
finalUrl += (if (isAnimated) ".gif" else ".png") + "?quality=lossless"
```
or
```kt
finalUrl += if (isAnimated) ".gif" else ".png"
finalUrl += "?quality=lossless"
```

Reply:
oh ok thanks